app/src/main/res/values/strings.xml

```diff
@@ -341,4 +339,9 @@
     <string name="tor_add_repo">Add Repository</string>
 
     <string name="nfc_type_not_supported">Unsupported NFC type</string>
+
+    <string name="help_dialog_scanPaymentRequest">QR-Codes containing the following are valid:\n\n- Bitcoin Address\n- Bitcoin Payment Request\n- Lightning Payment Request</string>
+    <string name="help_dialog_scanNodePublicKey">QR-Codes containing the following are valid:\n\n- pubkey\n- pubkey@host\n- pubkey@host:port</string>
+    <string name="help_dialog_scanConnectionInfo">QR-Codes containing the following are valid:\n\n- Lndconnect string (lndconnect://...)\n- BtcPay Server connection (config=...)\n- Full content of your BTCPay Server config file</string>
+    <string name="help_dialog_LightningVsOnChain">Lightning:\n\n+ instant\n+ low fees\n- not always possible\n\nPerfect for small transactions.\nUse whenever possible.\n\n\nOn-Chain:\n\n+ always possible\n- slower\n- higher fees\n\nGood for big transactions.\nUse this when Lightning is not possible.</string>
```

Comment:
If you haven't, maybe check with jimmy if the explanation of lightning vs onchain is fine regarding UX.

Reply:
Yes, good point, I'll talk with jimmy about it.